src/Mod/CAM/Path/Tool/assets/ui/preferences.py

```diff
@@ -0,0 +1,128 @@
+        self.form.setWindowTitle(translate("CAM_PreferencesAssets", "Assets"))
+
+        asset_path_widget = QtGui.QWidget()
+        main_layout = QtGui.QHBoxLayout(asset_path_widget)
+
+        # Create widgets
+        self.asset_path_label = QtGui.QLabel(translate("CAM_PreferencesAssets", "Asset Directory:"))
+        self.asset_path_edit = QtGui.QLineEdit()
+        self.asset_path_note_label = QtGui.QLabel(
+            translate(
+                "CAM_PreferencesAssets",
+                "Note: Select the directory that will contain the "
+                "Bit/, Shape/, and Library/ subfolders.",
+            )
+        )
+        self.asset_path_note_label.setWordWrap(True)
+        self.select_path_button = QtGui.QToolButton()
+        self.select_path_button.setIcon(QtGui.QIcon.fromTheme("folder-open"))
+        self.select_path_button.clicked.connect(self.selectAssetPath)
+        self.reset_path_button = QtGui.QPushButton(translate("CAM_PreferencesAssets", "Reset"))
+        self.reset_path_button.clicked.connect(self.resetAssetPath)
+
+        # Set note label font to italic
+        font = self.asset_path_note_label.font()
+        font.setItalic(True)
+        self.asset_path_note_label.setFont(font)
+
+        # Layout for asset path section
+        edit_button_layout = QtGui.QGridLayout()
+        edit_button_layout.addWidget(self.asset_path_label, 0, 0, QtCore.Qt.AlignVCenter)
+        edit_button_layout.addWidget(self.asset_path_edit, 0, 1, QtCore.Qt.AlignVCenter)
+        edit_button_layout.addWidget(self.select_path_button, 0, 2, QtCore.Qt.AlignVCenter)
+        edit_button_layout.addWidget(self.reset_path_button, 0, 3, QtCore.Qt.AlignVCenter)
+        edit_button_layout.addWidget(self.asset_path_note_label, 1, 1, 1, 1, QtCore.Qt.AlignTop)
+        edit_button_layout.addItem(
+            QtGui.QSpacerItem(0, 0, QtGui.QSizePolicy.Minimum, QtGui.QSizePolicy.Expanding),
+            2,
+            0,
+            1,
+            4,
+        )
+
+        main_layout.addLayout(edit_button_layout, QtCore.Qt.AlignTop)
+
+        self.form.addItem(asset_path_widget, translate("CAM_PreferencesAssets", "Assets"))
+
+    def selectAssetPath(self):
+        # Implement directory selection dialog
+        path = QtGui.QFileDialog.getExistingDirectory(
+            self.form,
+            translate("CAM_PreferencesAssets", "Select Asset Directory"),
+            self.asset_path_edit.text(),
+        )
+        if path:
+            self.asset_path_edit.setText(str(path))
+
+    def resetAssetPath(self):
+        # Implement resetting path to default
+        default_path = Path.Preferences.getDefaultAssetPath()
+        self.asset_path_edit.setText(str(default_path))
+
+    def saveSettings(self):
+        # Check path is writable, then call Path.Preferences.setAssetPath()
+        asset_path = pathlib.Path(self.asset_path_edit.text())
+        param = FreeCAD.ParamGet("User parameter:BaseApp/Preferences/Path")
```

Comment:
The correct parameter group should be BaseApp/Preferences/CAM

Reply:
Good catch! Done